coffee_cmd/src/main.rs

```diff
@@ -78,6 +78,19 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
+        CoffeeCommand::Search { plugin } => match coffee.search(&plugin).await {
+            Ok(val) => {
+                let val = val["search"].as_str().unwrap();
+                if val == "notfound" {
+                    term::markdown(&("unable to locate plugin ".to_owned() + &plugin));
```

Comment:
No I do not like this! You can return an error here!

Reply:
Does the changes I made suffice or should the function print the coffee error itself?

Reply:
The error is print at the line 97